web/themes/custom/usagov/scripts/ceoSubmission.js

```diff
@@ -56,7 +56,14 @@ const usps_translations = {
         "invalid-zip": "Por favor, escriba un código postal válido de 5 dígitos."
     }
 };
-let usps_content = usps_translations[document.documentElement.lang];
+
+let usps_content;
+if (usagovUSPSErrors) {
```

Comment:
I chose to test for backwards-compatibility by attempting to use the CEO tool with the custom twig content. I ran into an error here because usagovUSPSErrors isn't defined.